test/translate-test.js

```diff
@@ -12,6 +12,19 @@ describe('translateResponse', () => {
     customApiResponse = JSON.parse(resultContent);
   });
 
+  describe('create diagnostic report', () => {
+    it('should create diangostic report from custom API response', () => {
```

Comment:
```suggestion
    it('should create diagnostic report from custom API response', () => {
```

Reply:
Change to DiagnosticReport

Reply:
Nice! I see it was updated in https://github.com/ccsm-cds-tools/ccsm-cds-service/pull/27/commits/1e99515f3c50f01dc13a1a5c4342f2abcaa507d2.